### PR TITLE
bugfix - Connection pileup

### PR DESCRIPTION
**Pull request by kovacshuni**

### CODE REVIEW

**Comment by tosan88** [comment]
:+1:

**Comment by Zapadlo** [comment]
+1

File: main.go [hunk 1]
```diff
@@ -60,25 +60,31 @@ func main() {
 
 	app.Action = func() {
 		initLogs(os.Stdout, os.Stdout, os.Stderr)
-		transport := &http.Transport{Dial: proxy.Direct.Dial}
+		transport := &http.Transport {
+			Dial: proxy.Direct.Dial,
+			ResponseHeaderTimeout: 10 * time.Second,
+			MaxIdleConnsPerHost: 100,
```
Comment (Nesze) [inline main.go]: How does this number come up?

Is the idea that we have about 150-200 services then on a single host probably there isn't more then 100 running (probably 30-40)?

**Comment by Nesze** [comment]
+1

**Comment by BerniVarga** [comment]
+1

File: main.go (hunk 1 above)
Comment (kovacshuni) [inline main.go]: We had a discussion about this with Zoli.

**Pull request merged**